docs/public/content/admin/tutorials/registry.md

```diff
@@ -0,0 +1,49 @@
+# Push Docker images on a registry
+
+!!! warning "Official @@PROJECT_NAME@@ images"
+    @@PROJECT_NAME@@ does **not** officially publish any docker image on any public registry such as Docker hub or Quay.io. That said it provides a Bash script to push local images to any Docker registry.
+
+## Pre-requisites
+
+- git
+- bash
+- docker
+
+## Generate @@PROJECT_NAME@@ images
+
+In order to generate the Docker images to upload on the registry you should generate @@PROJECT_NAME@@'s JAVA archives. This can be done following the first part of the [Deploy on Docker tutorial](../docker#build-docker-images). To briefly summarize you should build and run an image which will populate your local directories with the @@PROJECT_NAME@@ JARs and WARs. In the end of this preliminary step you should have your images locally, this can be tested with the command `docker images | grep vulnerability-assessment-tool`.
```

Comment:
In order to generate the Docker images to upload to a local registry, you should generate @@PROJECT_NAME@@'s Java archives. This can be done following the first part of the [Deploy on Docker tutorial](../docker#build-docker-images). To briefly summarize you should build and run an image which will populate your local directories with the @@PROJECT_NAME@@ JARs and WARs. In the end of this preliminary step you should have your images locally, this can be tested with the command `docker images | grep vulnerability-assessment-tool`.